include/avif/avif.h

```diff
@@ -1107,6 +1107,11 @@ AVIF_API void avifEncoderSetCodecSpecificOption(avifEncoder * encoder, const cha
 
 // Helpers
 AVIF_API avifBool avifImageUsesU16(const avifImage * image);
+// channel can be an avifChannelIndex or 3 for alpha.
```

Comment:
I discussed this pull request with Joe. Joe suggested that we add `AVIF_CHAN_A` to the `avifChannelIndex` enum. Then we can declare the `channel` parameter as `avifChannelIndex channel` and get rid of this comment.

I don't quite remember why I objected to this when you suggested this. It may have to do with the comment in the `avifChannelIndex` enum definition:

```
typedef enum avifChannelIndex
{
    // These can be used as the index for the yuvPlanes and yuvRowBytes arrays in avifImage.
    AVIF_CHAN_Y = 0,
    AVIF_CHAN_U = 1,
    AVIF_CHAN_V = 2
} avifChannelIndex;
```

Joe suggests adding a blank line and a comment, something like:

```
typedef enum avifChannelIndex
{
    // These can be used as the index for the yuvPlanes and yuvRowBytes arrays in avifImage.
    AVIF_CHAN_Y = 0,
    AVIF_CHAN_U = 1,
    AVIF_CHAN_V = 2,

    // This may not be used in yuvPlanes and yuvRowBytes, but is available for use with avifImagePlane().
    AVIF_CHAN_A = 3
} avifChannelIndex;
```

Reply:
Done. In C++ files it is fine because we can use `for (avifChannelIndex c : {AVIF_CHAN_Y, AVIF_CHAN_U, AVIF_CHAN_V, AVIF_CHAN_A})`, but C files will need `for (int i = 0; i < 4; ++i)` and a `(avifChannelIndex)c` cast to call `avifImagePlane()`.

Reply:
> I would also change the for loops back to `for (int c = 0; c < 4; c++)`.

Why? I find `for (avifChannelIndex c : {AVIF_CHAN_Y, AVIF_CHAN_U, AVIF_CHAN_V, AVIF_CHAN_A})` longer (unfortunately) but type-safe and easier to understand.

Reply:
Using `for (int c = 0; c < 4; c++)` is optional. I didn't realize you prefer the new form. (I thought you may have used the new form to avoid the `avifChannelIndex` type cast.)

Reply:
For what it's worth, when I saw Yannis' comment about the loops, I had a very similar reaction to Wan-Teh (that we should leave the argument as `int`). This kind of function is almost always called in a loop from C, and it isn't worth the annoying casts. We should leave the function signature alone.